test/testdata/namer/fuzz_type_template_overwrite.rb.symbol-table-raw.exp

```diff
@@ -4,7 +4,7 @@ class <C <U <root>>> < <C <U Object>> () @ (... removed core rbi locs ..., Loc {
       argument <blk><block> @ Loc {file=test/testdata/namer/fuzz_type_template_overwrite.rb start=??? end=???}
   class <C <U A>> < <C <U Object>> () @ Loc {file=test/testdata/namer/fuzz_type_template_overwrite.rb start=2:1 end=2:8}
     static-field <C <U A>><C <U B>> -> AliasType { symbol = <S <C <U A>> $1><C <U B>> } @ Loc {file=test/testdata/namer/fuzz_type_template_overwrite.rb start=4:3 end=4:20}
-    static-field <C <U A>><M <C <U B>> $1> -> Integer(1) @ Loc {file=test/testdata/namer/fuzz_type_template_overwrite.rb start=3:3 end=3:4}
+    static-field <C <U A>><M <C <U B>> $1> @ Loc {file=test/testdata/namer/fuzz_type_template_overwrite.rb start=3:3 end=3:4}
```

Comment:
This test has two static fields:

```
  B = 1
  B = type_template
```

Since we insert symbols into the tree _after_ mangling, the last symbol wins -- changing this output.